kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build_download.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+#
+# This script downloads the latest WebLogic Deploy Tool and WebLogic Image Tool 
+# to the WORKDIR directory.
+#
+# Expects the following env vars to already be set:
+#    
+#    WORKDIR 
+#      working directory for the sample with at least 10g of space
+#
+# Optional env vars:
+#    http_proxy https_proxy
+#      If running behind a proxy, then set as needed to allow curl access to github.com.
+#
+#    DOWNLOAD_WDT DOWNLOAD_WIT
+#      Default to 'when-missing'. Set to 'always' to force download even
+#      if local installer zip is missing.
+#
+#    WDT_INSTALLER_URL WIT_INSTALLER_URL
+#      Defaults to 'https://github.com/oracle/weblogic-deploy-tooling/releases/latest'
+#      and 'https://github.com/oracle/weblogic-image-tool/releases/latest' respectively.
+#
+#      To override an installer URL, export the URL env to point to a specific zip file, for example:
+#      export WDT_INSTALLER_URL=https://github.com/oracle/weblogic-deploy-tooling/releases/download/weblogic-deploy-tooling-1.7.0/weblogic-deploy.zip
+#
+
+set -o pipefail
+
+set -eu
+
+SCRIPTDIR="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
+echo "@@ Info: Running '$(basename "$0")'."
+
+cd ${WORKDIR}
+
+DOWNLOAD_WIT=${DOWNLOAD_WIT:-when-missing}
+DOWNLOAD_WDT=${DOWNLOAD_WDT:-when-missing}
+WDT_INSTALLER_URL=${WDT_INSTALLER_URL:-https://github.com/oracle/weblogic-deploy-tooling/releases/latest}
```

Comment:
Conflicted... latest is good for a sample, but I'd really like to specify my versions more easily.

Reply:
You already can specify a specific version by specifying the full URL to the installer - instructions are in the description comments above for WDT_INSTALLER_URL and WIT_INSTALLER_URL.

Reply:
@rjeberhard Please resolve this if you think it's OK. Otherwise provide more feedback. Thanks.